packages/tsx-editor/src/components/Editor.tsx

```diff
@@ -64,7 +63,7 @@ export const Editor: React.FunctionComponent<IEditorProps> = (props: IEditorProp
       }
 
       if (currDebounceTime) {
-        debounceTimeout = setTimeout(() => currOnChange(model.getValue()), currDebounceTime);
+        debounceTimeout = (setTimeout(() => currOnChange(model.getValue()), currDebounceTime) as unknown) as number;
```

Comment:
I think you can do `window.setTimeout` to get the correct typing here and not have to do that `as unknown as number` cast.

Reply:
Good point, thanks!